packages/constants/src/constants.ts

```diff
@@ -133,6 +133,12 @@ export const MC_API_PROXY_TARGETS = {
 } as const;
 export type TMcApiProxyTargets = typeof MC_API_PROXY_TARGETS[keyof typeof MC_API_PROXY_TARGETS];
 
+export type TTokenRetryGraphQlTarget =
```

Comment:
I would prefer to keep this in the app shell package and not leak it in constants, as it's really something internal to AppShell.
Sorry 😬

Reply:
See last commit.